src/EFCore.Abstractions/UnicodeAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.EntityFrameworkCore
+{
+    /// <summary>
+    ///     Configures the property as capable of persisting unicode characters. Can only be set on <see cref="string" /> properties.
```

Comment:
```suggestion
    ///     Configures the property as capable of persisting unicode characters.
```
It can be set on property which maps to string like type in database using a converter.